Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUCreateImpl.cpp

```diff
@@ -42,7 +42,7 @@ WTF_WEAK_LINK_FORCE_IMPORT(wgpuCreateInstance);
 
 namespace PAL::WebGPU {
 
-RefPtr<GPU> create(ScheduleWorkFunction&& scheduleWorkFunction)
+RefPtr<GPU> create(ScheduleWorkFunction&& scheduleWorkFunction, mach_port_t webProcessID)
```

Comment:
It's probably not "web process id" if you are implementing a general purpose webgpu implementation.
I've been calling these "resource owner"
Being pedantic, the type is not `mach_port_t` but a `task_id_token_t`.

The value cannot be held willy-nilly, it's a refcounted object with a lifetime.
The best we have currently for holding this is WebCore::ProcessIdentity, if you can depend on WebCore::. Otherwise it would be WTF::MachSendRight.